tests/performance/scripts/runperf.sh

```diff
@@ -32,7 +32,7 @@ cat >config.json<<END
 {
         "image_url": ["nginx", "hello-world"],
         "k8smasterforkubeedge": "$K8SMasterForKubeEdge",
-        "node_num": 10,
+        "node_num": 500,
```

Comment:
It will be great that we will be running our performance tests against 500 edge nodes :)

Reply:
Thanks @sids-b The testing is in progress